### PR TITLE
install_config/install: minor improvements

### PR DESCRIPTION
**Pull request by php-coder**
@openshift/team-documentation PTAL

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@adellape - PTAL.

**Comment by php-coder** [comment]
@adellape PTAL

**Comment by php-coder** [comment]
@openshift/team-documentation ping

File: install_config/install/host_preparation.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ If you are using https://www.vagrantup.com[Vagrant] to run {product-title}, you
 do not need to go through the following sections. These changes are only
 necessary when you are setting up the host yourself. If you are using Vagrant,
 see the
-https://github.com/openshift/origin/blob/master/CONTRIBUTING.html#develop-on-virtual-machine-using-vagrant[Contributing
+https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc#develop-on-virtual-machine-using-vagrant[Contributing
```
Comment (ahardin-rh) [inline install_config/install/host_preparation.adoc]: you need to put the following in front of the path:

link:

File: install_config/install/host_preparation.adoc [hunk 2]
```diff
@@ -570,7 +570,7 @@ drwx------. 2 root root    6 Nov 24 00:12 secrets
 ====
 
 See Docker's documentation for additional information on how to
-http://docs.docker.com/engine/reference/logging/overview/#the-json-file-options[Configure
+https://docs.docker.com/engine/admin/logging/overview/#/options[Configure
```
Comment (ahardin-rh) [inline install_config/install/host_preparation.adoc]: link:

**Comment by ahardin-rh** [comment]
@php-coder just a few comments. We will also need the changes to be squashed into one commit for easy cherry-picking and publishing. Thanks!

**Comment by php-coder** [comment]
@ahardin-rh Thank you for a quick response! I've fixed the comments and squashed commits.

**Comment by ahardin-rh** [comment]
@php-coder Thank you! Does this apply to 3.4?

**Comment by php-coder** [comment]
> Does this apply to 3.4?

Yes, most likely it does.

**Comment by ahardin-rh** [comment]
Thank you! Confirmed that this also fixes a link in 3.3.

**Comment by ahardin-rh** [comment]
Minor edits. No revision history needed.

**Pull request merged**